framework/src/main/java/org/motechproject/uitest/page/DataServicesPage.java

```diff
@@ -42,13 +50,47 @@ public String createNewEntity(String entityName) throws InterruptedException {
     }
 
     /**
+     * Method that goes to schema editor page and enters entity
+     * @param entityName name of the entity we want to edit
+     */
+    public void goToEditEntity(String entityName) throws InterruptedException {
+        clickWhenVisible(DATA_SERVICES_BUTTON);
+        clickWhenVisible(By.id(String.format("edit_%s", entityName)));
+    }
+
+    /**
```

Comment:
do not start javadoc with word 'method', in this case begin with 'Adds'.

Add fieldDisplayName field to javadocs